### DISCUSSION TITLE
New NewImageFromFile not work

### DISCUSSION THREAD

**Discussion by illua1** [hajimehoshi/ebiten#1441] [Q&A]
After changes in the function, it stopped working. For any address to the image, it returns only an nil
```go
img,_,_ := ebitenutil.NewImageFromFile( "testImage2.png", ebiten.FilterDefault ); // works!
img,_,_ := ebitenutil.NewImageFromFile( "testImage2.png" ); // NO works// only nil```

**Comment by hajimehoshi** [comment]
I think you are using Ebiten v1, not v2. Did you import `github.com/hajimehoshi/ebiten` instead of `github.com/hajimehoshi/ebiten/v2`?

**Reply by illua1** [reply]
> I think you are using Ebiten v1, not v2. Did you import `github.com/hajimehoshi/ebiten` instead of `github.com/hajimehoshi/ebiten/v2`?

maybe I'm a beginner, and not familiar with other versions, I imported `github.com/hajimehoshi/ebiten`
however, by the same team, a month earlier, I was putting together another project. Having found new services in the library folders, I compared the changes in the function. And also copied the previous version to the working directory.
strange that ` my get` doesn't see `/v2`
The version in which
```go
img, _, _: = ebitenutil.NewImageFromFile ("testImage2.png", ebiten.FilterDefault);
```
works great, sad that the stream is lost when I press the console

**Reply by hajimehoshi** [reply]
> maybe I'm a beginner, and not familiar with other versions, I imported github.com/hajimehoshi/ebiten

To use the latest version, you should use `github.com/hajimehoshi/ebiten/v2`. Note that you cannot mix v1 and v2 in one project.

**Reply by illua1** [reply]
> > maybe I'm a beginner, and not familiar with other versions, I imported github.com/hajimehoshi/ebiten
> 
> To use the latest version, you should use `github.com/hajimehoshi/ebiten/v2`. Note that you cannot mix v1 and v2 in one project.

```go
_,img,_ := ebitenutil.NewImageFromFile( `icon test.png` );
ebiten.SetWindowIcon( []image.Image{img,} );
```
I mean
checked it in the fastest way
this feature in v2 does not work as expected

```go
_,img,_ := ebitenutil.NewImageFromFile( `icon test.png`, ebiten.FilterDefault  );
ebiten.SetWindowIcon( []image.Image{img,} );
```
the previous version is working properly
but it is in the version in which the game freezes the flow when unselecting

**Reply by hajimehoshi** [reply]
Could you show me the import section or the entire file in your project?
